docs/dev/guide/contract-verification.md

```diff
@@ -72,10 +72,35 @@ To verify the contract, enter the following details;
 <br>
 **Note:** You may need to flatten your solidity code if it utilizes a library or inherits dependencies from another contract. 
 <br> 
-We recommend using the[truffle flattener](https://github.com/NomicFoundation/truffle-flattener) or the [POA solidity flattener](https://github.com/poanetwork/solidity-flattener).
+We recommend using the [truffle flattener](https://github.com/NomicFoundation/truffle-flattener) or the [POA solidity flattener](https://github.com/poanetwork/solidity-flattener).
 
 - Constructor arguments: If Constructor Arguments are required by the contract, you should add them here, but in [ABI hex encoded form](https://solidity.readthedocs.io/en/develop/abi-spec.html).
-- Click the **Verify Smart Contract button**.
+  
+<br>
+
+**Note:** To obtain the Constructor arguments, after deploying your contract, include this 
+<br>
+
+```
+console.log("constructor args:" + greeterContract.interface.encodeDeploy([greeting]));
```

Comment:
This is specifically for the greeterContract that we have in our Hello World tutorial. It won't always work

Reply:
we also need to explain that then the args will be output on the screen on contract deployment

Reply:
Noted.